.devcontainer/devcontainer.json

```diff
@@ -8,7 +8,11 @@
     "ghcr.io/devcontainers/features/python:1.6.3": {
       "version": "3.12.0"
     },
-    "ghcr.io/devcontainers/features/aws-cli:1": {}
+    "ghcr.io/devcontainers/features/aws-cli:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2.12.0": {
+      "version": "27.0.3",
+      "moby": false
+    }
```

Comment:
This change a at least another one are already in `dev`. This appears here because someone GitHub fails to refresh the changed files.

Reply:
I rebased this feature branch with the base branch and `git push --force`.